net-core/docs/kickoff/README.md

```diff
@@ -0,0 +1,51 @@
+NET CORE Kickoff guide
+-------------------------
+
+The following guide explains how to make a NET Core project kickoff from scratch.
+
+## Requirements
+
+Before you start with this step by step guide you will need the following:
+* Access to the Github repository where the code of the kickoff will be hosted.
+
+## Kickoff
+
+The following steps will help you have a NET Core project ready for new features. We are going to use an example project with the name `app-name`. Everytime you see `App Name` in this guide, you should replace it with your project name. It is important to respect all the name conventions.
```

Comment:
I'd remove 'you should' from the second sentence (it's actually a 'must' because it shouldn't be done any other way, right?)

Reply:
yes :)